normalization-class-2021/MiniClass.tex

```diff
@@ -245,7 +245,7 @@ \subsection{Call-by-name reduction}
       {\infer[\beta]{(\lambda\,x.\, \lambda\,y.\, \lambda z.\, (y\ x))\ (\textit{id}\ \textit{id})\ \leadsto\
           (\lambda\,y.\, \lambda\,z.\, (y\ (\textit{id}\ \textit{id})))}{\ }}
       \]
-\noindent This derivation proves the single reduction step (at the bottom of the derivation)
+\noindent This derivation proves the single reduction step (at the bottom of the derivation):
```

Comment:
I actually do not want a colon here, because the sentence is directly referring to the reduction step (as if I wrote "These notes cover the book 'How to Peel Apples'").

Reply:
Thanks, that nuance makes sense. Reverted this change.